### PR TITLE
Bioconductor/sweave2rmd: Conversion DirichletMultinomial.Rnw to Rmd

### PR DESCRIPTION
**Pull request by Khadeeejah**
Bioconductor/sweave2rmd
My name is Khadijah an Outreachy applicant.  This is my second contribution I was assigned this [issue](https://github.com/mtmorgan/DirichletMultinomial/blob/master/vignettes/DirichletMultinomial.Rnw) on the project board,

This pr is not that great, but i was hoping i would be able to record my contribution before the deadline and also get your feedback and corrections. Thank you for always being patient and helpful
below is the
 
[HTML ]( https://rpubs.com/khadeeejah01/1023080)

[PDF](https://bioconductor.org/packages/release/bioc/vignettes/DirichletMultinomial/inst/doc/DirichletMultinomial.pdf)

cc @jwokaty

### CODE REVIEW

**Comment by jwokaty** [comment]
@Khadeeejah Thanks for this PR. I'm going to provide some initial feedback so that you can address a few issues while you wait for a full review.

File: DESCRIPTION
Comment (jwokaty) [inline DESCRIPTION:7]: Martin is both the author and maintainer of the package. For his `person` entry. the role should be `role=c("aut", "cre")` and it must also include an email attribute since he is the maintainer as required by Bioconductor standards.

File: DESCRIPTION
Comment (jwokaty) [inline DESCRIPTION:11]: - [ ] Remove lines 10-11 since Martin is the maintainer.
- [ ] Replace the comma at the end of line 9 with a closing parenthesis.

File: vignettes/DirichletMultinomial.Rmd
Comment (jwokaty) [inline vignettes/DirichletMultinomial.Rmd:31]: `lattice` is CRAN package, so we should generate the link with

    `r CRANpkg("lattice")`

And I believe `parallel` is in base R so you can use

    `r Rpackage("parallel")`

File: vignettes/DirichletMultinomial.Rmd
Comment (jwokaty) [inline vignettes/DirichletMultinomial.Rmd:28]: Be careful with inline code; if it's split on 2 lines, it will just be printed as text in the knitted file. The following must be on one line:

    `r Biocpkg("DirichletMultinomial")`

File: vignettes/DirichletMultinomial.Rmd
Comment (jwokaty) [inline vignettes/DirichletMultinomial.Rmd:13]: Capitalize `R` in `References.bib

File: vignettes/DirichletMultinomial.Rmd
Comment (jwokaty) [inline vignettes/DirichletMultinomial.Rmd:42]: Replace brackets followed by `{.sans-serif}` with backticks such as

    `R`

File: vignettes/DirichletMultinomial.Rmd
Comment (jwokaty) [inline vignettes/DirichletMultinomial.Rmd:76]: Remove extra spaces such as these.

File: vignettes/DirichletMultinomial.Rmd
Comment (jwokaty) [inline vignettes/DirichletMultinomial.Rmd:125]: I believe it should be `min-laplace.pdf`.

**Review by jwokaty — CHANGES_REQUESTED** [review]
This isn't a complete review. I just wanted to provide some initial feedback so you can get started on changes. I've noted a few inline issues to address. Additionally, do the following:

- [ ] Remove the following files: .Rbuildignore, .gitignore, and DirichletMultinomial.Rproj
- [ ] Your .Rmd doesn't currently knit
- [ ] Remove extra spaces. Generally, most things are separated by a blank line.

**Comment by Khadeeejah** [comment]
Thank you @jwokaty  I'll get started with them asap

**Comment by mcarlsn** [comment]
@Khadeeejah Thanks so much for your work on this conversion. We're in the process of organizing the Sweave2Rmd project board post-Outreachy contribution period ending and I wanted to follow-up with you on this vignette. Are you interested in completing the conversion? If not, no worries - we can pass it along to someone else - but I wanted to check with you first. Let us know!

**Comment by Khadeeejah** [comment]
> @Khadeeejah Thanks so much for your work on this conversion. We're in the process of organizing the Sweave2Rmd project board post-Outreachy contribution period ending and I wanted to follow-up with you on this vignette. Are you interested in completing the conversion? If not, no worries - we can pass it along to someone else - but I wanted to check with you first. Let us know!

Hi, sorry for being MIA I've been sick, just getting better I intend to make a new PR tonight or Tommorow morning.
I would finish the task.

**Comment by mcarlsn** [comment]
> 

I'm so sorry to hear that you've been sick! I hope you feel better soon.

**Comment by jwokaty** [comment]
@mtmorgan This PR has likely been abandoned and can be closed. I'll make it available for conversion again.

**Pull request closed without merging**
